roles/openobserve/defaults/main.yml

```diff
@@ -0,0 +1,145 @@
+##########################################################################
+# Title:         Sandbox: openobserve | Default Variables                #
+# Author(s):     salty, Thundernerd                                      #
+# URL:           https://github.com/saltyorg/Sandbox                     #
+# --                                                                     #
+##########################################################################
+#                   GNU General Public License v3.0                      #
+##########################################################################
+---
+################################
+# Basics
+################################
+
+openobserve_name: openobserve
+
+################################
+# Paths
+################################
+
+openobserve_paths_folder: "{{ openobserve_name }}"
+openobserve_paths_location: "{{ server_appdata_path }}/{{ openobserve_paths_folder }}"
+openobserve_paths_folders_list:
+  - "{{ openobserve_paths_location }}"
+
+################################
+# Web
+################################
+
+openobserve_web_subdomain: "{{ openobserve_name }}"
+openobserve_web_domain: "{{ user.domain }}"
+openobserve_web_port: "5080"
+openobserve_web_url: "{{ 'https://' + (openobserve_web_subdomain + '.' + openobserve_web_domain
+                 if (openobserve_web_subdomain | length > 0)
+                 else openobserve_web_domain) }}"
+
+################################
+# DNS
+################################
+
+openobserve_dns_record: "{{ openobserve_web_subdomain }}"
+openobserve_dns_zone: "{{ openobserve_web_domain }}"
+openobserve_dns_proxy: "{{ dns.proxied }}"
+
+################################
+# Traefik
+################################
+
+openobserve_traefik_sso_middleware: ""
+openobserve_traefik_middleware_default: "{{ traefik_default_middleware }}"
+openobserve_traefik_middleware_custom: ""
+openobserve_traefik_certresolver: "{{ traefik_default_certresolver }}"
+openobserve_traefik_enabled: true
+openobserve_traefik_api_enabled: false
+openobserve_traefik_api_endpoint: ""
+
+################################
+# Docker
+################################
+
+# Container
+openobserve_docker_container: "{{ openobserve_name }}"
+
+# Image
+openobserve_docker_image_pull: true
+openobserve_docker_image_tag: "latest"
+openobserve_docker_image: "public.ecr.aws/zinclabs/openobserve:{{ openobserve_docker_image_tag }}"
+
+# Ports
+openobserve_docker_ports_defaults: []
+openobserve_docker_ports_custom: []
+openobserve_docker_ports: "{{ openobserve_docker_ports_defaults
+                         + openobserve_docker_ports_custom }}"
+
+# Envs
+openobserve_docker_envs_default:
+  TZ: "{{ tz }}"
+  PUID: "{{ uid }}"
+  PGID: "{{ gid }}"
+  ZO_ROOT_USER_EMAIL: "root@example.com"
+  ZO_ROOT_USER_PASSWORD: "Complexpass#123"
```

Comment:
[HELP]

Would love to know of a way to change this so that it is somewhere in the settings? Maybe a standard email/password that is already configured by the user?

You cannot change these once the container has been started in the user management panel which is very unfortunate.

Reply:
https://github.com/saltyorg/Sandbox/blob/b8f2f4b7c3af77fddba29573d9b599350b04a477/roles/pgadmin/defaults/main.yml#L67

Here is an example of how to pull in the email and password from the Saltbox settings.yml file

Reply:
Works like a charm, cheers!